File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> ind;
        int n = nums.size();
        for(int i = 0; i < n; i++) {
            if(ind.count(target - nums[i])) {
                return {ind[target - nums[i]], i};
            }
            else
                ind[nums[i]] = i;
        }

        return vector<int>();
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        s = set(nums)
        res = 0
        for i in s:
            if i - 1 not in s:
                j = i
                while j in s:
                    j += 1
                res = max(res, j - i)
        
        return res